alchemiscale/base/client.py

```diff
@@ -201,13 +329,18 @@ def _query_resource(self, resource, params=None):
 
     @_retry
     @_use_token
-    def _get_resource(self, resource, params=None):
+    def _get_resource(self, resource, params=None, compress=False):
```

Comment:
Did we want to compress by default here and below? Up to you. It seemed set to default to `True` on a lot of the `interface/client.py` methods.

Reply:
My thinking here is to keep compression on these private methods opt-in. Not all `post` and `get` calls will benefit from compression, especially for tiny requests/responses, so making it something we enable as the default on specific user-facing methods made the most sense to me.